ddpui/utils/redis_client.py

```diff
@@ -0,0 +1,18 @@
+import os
+from redis import Redis
+
+
+class RedisClient:
```

Comment:
Will this design create multiple connections from the app to Redis? Do we need to worry at all about managing multiple connections?

Reply:
My thinking is it won't because the class method only creates a new connection instance if no previous instance exists else it just returns the existing instance.